packages/lit-html/package.json

```diff
@@ -155,6 +152,127 @@
     "!/development/test/",
     "/directives/"
   ],
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:rollup",
+        "build:ts:types",
+        "build:version-stability-test"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc --build --pretty",
+      "clean": "if-file-deleted",
+      "files": [
+        "src/**/*.ts",
+        "tsconfig.json"
+      ],
+      "output": [
+        "development",
+        "tsconfig.tsbuildinfo",
+        "tsconfig.polyfill-support.tsbuildinfo"
+      ]
+    },
+    "build:ts:types": {
+      "command": "treemirror development . \"**/*.d.ts{,.map}\"",
+      "dependencies": [
+        "../internal-scripts:build",
+        "build:ts"
+      ],
+      "files": [],
+      "output": [
+        "*.d.ts{,.map}",
+        "directives/*.d.ts{,.map}"
+      ]
+    },
+    "build:rollup": {
+      "command": "rollup -c",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": [
+        "async-directive.js{,.map}",
+        "directive-helpers.js{,.map}",
+        "directive.js{,.map}",
+        "experimental-hydrate.js{,.map}",
+        "lit-html.js{,.map}",
+        "polyfill-support.js{,.map}",
+        "private-ssr-support.js{,.map}",
+        "static.js{,.map}",
+        "directives/*.js{,.map}"
+      ]
+    },
+    "build:version-stability-test": {
+      "command": "rollup -c rollup-version-stability-test.config.js",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup-version-stability-test.config.js",
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": [
+        "version-stability-build"
+      ]
+    },
+    "checksize": {
+      "command": "rollup -c --environment=CHECKSIZE",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
+      ],
+      "output": []
+    },
+    "check-version": {
+      "command": "node scripts/check-version-tracker.js",
+      "dependencies": [
+        "build:ts"
```

Comment:
This script checks the source, so it doesn't need to depend on `build:ts`

Reply:
Good call, done.